efel/api.py

```diff
@@ -99,6 +99,7 @@ def reset():
     setStrSetting("current_base_mode", "mean")
     setDoubleSetting("precision_threshold", 1e-10)
     setDoubleSetting("sahp_start", 5.0)
+    setIntSetting("ignore_first_spike", 1)
```

Comment:
Is this a bool? (i.e. 0 == false, 1 is true)?. If yes, then let's limit this to these two values in the code below, and let's add it to the documentation